spec/shore/client/tokens/access_token_spec.rb

```diff
@@ -0,0 +1,132 @@
+require 'spec_helper'
+
+describe Shore::Client::Tokens::AccessToken do
```

Comment:
Let's use the strictest possible RSpec syntax. With that the outer `describe ...` must be `RSpec.describe ...`. All of the other calls to describe within the block don't need to be prefixed with `RSpec.` though. Take a look at the spec_helper.rb in the messaging service. I think this has all of the strictest settings. I copied it from Core. Core we upgraded from RSpec 2 to 3 and the strict settings made it easier to upgrade.